### PR TITLE
Feature/add basic cli

### PR DESCRIPTION
**Pull request by vid**
## PR description

Adds a basic command line interface. No tests are provided since it just invokes the tested lib and we aren't here to test the runtime.

## Checklist

- [ x] - Added tests
- [ x] - Ran `npm test`, ensuring linting passes
- [x ] - Adjust README documentation if relevant

### CODE REVIEW

File: src/jsonp-cli.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    process.exit(1);
+}
+
+function runQuery(json, path) {
+    const result = jsonpath({
+        json,
+        path,
+    });
+
+    // eslint-disable-next-line no-console
```
Comment (brettz9) [inline src/jsonp-cli.js:24]:
```suggestion
    // eslint-disable-next-line no-console -- CLI
```

Like to always give a reason for disabling...

File: src/jsonp-cli.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    console.log("result", result);
```
Comment (brettz9) [inline src/jsonp-cli.js:25]:
```suggestion
    console.log(result);
```

I think by removing the "result" string, a user could more easily pipe the results into a file or something...

File: package.json [hunk 3]
```diff
@@ -3,6 +3,9 @@
   "name": "jsonpath-plus",
   "version": "8.0.0",
   "type": "module",
+  "bin": {
+    "jsonp-cli": "./dist/jsonp-cli.js"
```
Comment (brettz9) [inline package.json:7]:
```suggestion
    "jsonp-cli": "./dist/jsonp-cli.js",
    "jsonpath": "./dist/jsonp-cli.js"
```

I think it's easier to remember with at least one more directly evoking the package name

**Review by brettz9 — CHANGES_REQUESTED** [review]
I think with the suggested changes and an update to the build file, we'd be ready.

**Comment by vid** [comment]
Hi, makes sense. I think it's better not to have two bin options so went with the original package name, but happy to make further improvements. Thanks!

**Comment by brettz9** [comment]
I'm thinking maybe we should add a `jsonpath-plus` binary too so that people can use the package name (as you originally did in the docs), as well as the shorter `jsonpath`.

**Comment by vid** [comment]
Oh you're right, I just added the package name as a bin mapping.

**Comment by brettz9** [comment]
I changed to avoid a dist file (I think CLI users can generally use the latest version) and linted the file. If you want to take a look, I think it should be ready after that.

**Comment by vid** [comment]
Makes sense. For completeness (…) I added a `files` directive.

**Review by brettz9 — CHANGES_REQUESTED** [review]

**Comment by vid** [comment]
Oh! I wasn't aware of that approach, reverted now.

**Comment by brettz9** [comment]
So do the changes I added look ok?

**Comment by vid** [comment]
Yes, it all "lgtm." Thanks!

**Review by brettz9 — APPROVED** [review]

**Comment by brettz9** [comment]
Thanks for the PR! Released as part of v8.1.0

**Comment by vid** [comment]
Thanks for guiding me through it.

**Pull request merged**